autobuild/tfchain.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -ex
+
+apt-get update
+apt-get install git gcc wget -y
+
+# make output directory
+ARCHIVE=/tmp/archives
+FLIST=/tmp/flist
+mkdir -p $ARCHIVE
+
+# install go
+wget https://dl.google.com/go/go1.10.linux-amd64.tar.gz
+tar -C /usr/local -xzf go1.10.linux-amd64.tar.gz
+mkdir -p /root/go
+export GOPATH=/root/go
+export PATH=$PATH:/usr/local/go/bin:$GOPATH/go/bin
+
+mkdir -p /root/go/src/github.com/threefoldfoundation
+cp -ar /tfchain /root/go/src/github.com/threefoldfoundation/tfchain
+
+TFCHAIN=$GOPATH/src/github.com/threefoldfoundation/tfchain
+TFCHAIND=$TFCHAIN/cmd/tfchaind
+TFCHAINC=$TFCHAIN/cmd/tfchainc
+
+pushd $TFCHAIND
+go build -ldflags "-linkmode external -extldflags -static" -o $FLIST/bin
```

Comment:
Whats the reason behind these linker flags? As far as I know, this is a 100% go project, so why is there external linking?

Reply:
Also, as these are supposedly production binaries, use the -s and -w linker flags to strip debug symbols, thus reducing size

Reply:
when I just did `go build` then `ldd` the binaries, there were some dependencies, and hence I needed it to be static binaries to avoid copying the libs.

Reply:
This autobuild automatically upload flist to the hub.
To have flist working out-of-box, we need static binary. This make sens.

Reply:
Alright then, though the debug symbol tables should still be stripped imo